find_tag:
  timeout: 5.0
  stale_tag_time: 1.0

go_to_pose:
  controller_type: strafe
  x_kP: 1.25
  x_kI: 0.001
  x_kD: 0.05
  y_kP: 1.25
  y_kI: 0.001
  y_kD: 0.05
  theta_kP: 3.0
  theta_kI: 0.001
  theta_kD: 0.05
  ramsete_b: 2.0
  ramsete_zeta: 0.7
  loop_rate: 50.0
  global_frame: "odom"
  settling_time: 0.5

move_base_namespace: /move_base
base_frame: base_link
global_frame: map

shuffle_until_charging:
  is_charging_cooldown: 5.0
  speed: 0.15
  loop_rate: 10.0
  interval: 0.4
